src/main/service/statusDescriptionService.ts

```diff
@@ -0,0 +1,34 @@
+import {SearchDescriptionActions} from '../resources/actions/searchDescriptionActions';
+import {InputFilterService} from '../service/inputFilterService';
+
+export class StatusDescriptionService {
+  private static generateAlphabetObject(): object {
```

Comment:
we dont need a new service for this, just use the one that already exists

Reply:
reuse some of the code. It is not possible to use all the code which already exists because we are creating new object on the base on json data. And json data attributes are different for each data type.